src/interfaces/CommonTypes.ts

```diff
@@ -56,3 +56,5 @@ export interface ApiQuery {
   query: string;
   advancedSearch: AdvancedSearch | null;
 }
+
+export type Theme = Record<string, unknown>;
```

Comment:
Can we change the name here? For instance, `UnknownDictionary` or something like this

Reply:
Oh, nice catch, I was a bit fast with this one. I think a good solution here is to instead do:
```
export type AnyIfEmpty<T extends Record<string, unknown>> = keyof T extends never ? any : T;
export type Theme = Record<string, unknown>;
```

Reply:
I'm not sure that `Record<string, unknown>` can be named as `Theme`, right? Cause `Theme` should be more specific, I believe

Reply:
In my opionion it's good to have types for things even though they are just an alias for something primitive. It makes it easier to add more specific typing in the future if we want. I don't look at `Theme` as a general concept though, it should only be used for `theme: ...` fields, so `UnknownDictionary` might still be relevant.

However I don't have too strong opinions here :)

Reply:
also, do you know why we have this `AnyIfEmpty` construct?

Reply:
I went for a middleground. So now we have:
```ts
export type UnknownDictionary = Record<string, unknown>;
export type AnyIfEmpty<T extends UnknownDictionary> = keyof T extends never
  ? any
  : T;
export type Theme = AnyIfEmpty<UnknownDictionary>;
```

I think this a good solution for now at least.

Reply:
Hmm, I like this solution!